main.go

```diff
@@ -399,6 +399,10 @@ func setupSupervisorControllers(ctx context.Context, controllerCtx *capvcontext.
 		return err
 	}
 
+	if err := controllers.AddMachineTemplateControllerToManager(ctx, controllerCtx, mgr, true, concurrency(1)); err != nil {
```

Comment:
Should we default to 10? Feels better than to default to "sequential" 1

Reply:
added a flag + defaulting to 10